docs/application/dotnet/guides/machine-learning/singleshot.md

```diff
@@ -0,0 +1,125 @@
+
+```C#
+/* Input Dimension: 3 * 224 * 224 */
+TensorsInfo in_info = new TensorsInfo();
+in_info.AddTensorInfo(TensorType.UInt8, new int[4] { 3, 224, 224, 1 });
+
+/* Output Dimension: 1001 for classification */
+TensorsData out_info = new TensorsInfo();
+out_info.AddTensorInfo(TensorType.UInt8, new int[4] { 1001, 1, 1, 1 });
+```
+
+<a name="load"></a>
+## Loading Neural Network Model and Configuring Runtime Environment
+
+1. Since the model file is located in the resource directory of your own application, you need to get its absolute path:
+
+    ```C#
+    string ResourcePath = Tizen.Applications.Application.Current.DirectoryInfo.Resource;
+    string model_path = ResourcePath + "models/mobilenet_v1_1.0_224_quant.tflite";
+    ```
+
+
+2. You can load the neural network model from storage and configure a runtime environment with the `Tizen.MachineLearning.Inference.SingleShot` class. The first parameter is the absolute path to the neural network model file. The remaining two parameters are the input and the output `TensorsInfo` instances. If there is an invalid parameter, `ArgumentException` is raised.
+
+
+
+    ```C#
+    /* Create SingleShot instance with model information */
+    SingleShot single = new SingleShot(model_path, in_info, out_info);
+    ```
+
+<a name="invoke"></a>
+## Invoking Neural Network Model using Input Data
+
+To invoke the neural network model, you need to create the `Tizen.MachineLearning.Inference.TensorsData` instance to pass the input data of the model. You can add various types of tensor data, which are already specified in the `TensorInfo` instance. However, the maximum size of `TensorsData` is 16. If the limit is exceeded, then `IndexOutOfRangeException` is raised. Input data is passed in a byte array format, byte[].
+
+```C#
+/* Input data for test */
+byte[] in_buffer = new byte[3 * 224 * 224 * 1];
+
+/* Set the input tensor data */
+TensorsData in_data = in_info.GetTensorsData();
+in_data.SetTensorData(0, in_buffer);
+```
+
+After preparing the input data, you can invoke the model and get the inference output result. The `SingleShot.Invoke()` method gets the input data to be inferred as a parameter and returns the `Tizen.MachineLearning.InferenceTensorsData` instance, which contains the inference result.
+
+```C#
+/* Invoke the model and get the inference result */
+TensorsData out_data = single.Invoke(in_data);
+```
+
+<a name="fetch"></a>
+## Fetching Inference Result
+
+After calling the `Invoke()` method of the `Tizen.MachineLearning.Inference.SingleShot` class,
+the `Tizen.MachineLearning.Inference.TensorsData` instance is returned as the inference result.
+The result can have multiple output data. Therefore, you have to fetch each data using the `GetTensorData()` method. If the limit is exceeded, then `IndexOutOfRangeException` is raised.
```

Comment:
This sentence means that the number of item is over its limit, then `IndexOutOfRangeException` is raised.
IMHO, most developers can understand its meaning since it is common case in this domain.

When my googling, "If the limit is exceeded" is more common than  "If the limit exceeds". So what about using the first one?